runtime/standalone/src/lib.rs

```diff
@@ -124,5 +124,8 @@ construct_runtime!(
         PolkadotBridge: pallet_grandpa_finality_verifier::<Instance1> = 130,
         KusamaBridge: pallet_grandpa_finality_verifier::<Instance2> = 131,
 
+        // Handy utilities
+        MaintenanceMode: pallet_maintenance_mode = 132,
```

Comment:
This version would need to make room for light clients, so prolly want something like 140

Reply:
Done